setup.cfg

```
@@ -36,6 +36,7 @@ python_requires = >=3.7
 include_package_data = True
 packages = find:
 install_requires =
+    pylangacq
```

It is best practice to specify a version, preferably using the 'compatible with notation':

```suggestion
    pylangacq~=0.18.0
```